### ISSUE TITLE
Allow arrow and TAB navigation in the file/folder dialogs 

### ISSUE THREAD

**Issue by alexballas** [fyne-io/fyne#2271]
### Is your feature request related to a problem? Please describe:
It will increase the file/folder dialog usability 

### Is it possible to construct a solution with the existing API?
Unknown

### Describe the solution you'd like to see:
Allow arrow and TAB navigation in the file/folder dialogs. Currently file selection and navigation is only available through mouse events

**Comment by andydotxyz** [comment]
Agreed. This may relate to our multi-select work as file dialog does not currently have selected state (as we just tap to open).
Let's see if we can get this in to next release.

**Comment by Jacalz** [comment]
I agree that this is a great addition and I think we might want to mimic what other file managers do in this regard. It seems that Gnome Nautilus not really allows to change the selected file using tab, only using arrows. Then enter/return can be used to either select a file or go into a new folder.

**Comment by andydotxyz** [comment]
Agreed - tab moves between the file panel and other elements of the UI

**Comment by alexballas** [comment]
Any plans to support "Ctrl + A", "Shift + click", "Ctrl + click" etc, or are you thinking just simple arrow navigation as a first step?

**Comment by Jacalz** [comment]
> Any plans to support "Ctrl + A", "Shift + click", "Ctrl + click" etc, or are you thinking just simple arrow navigation as a first step?

I don't think there is any reason to add "Ctrl + A" given that we don't support selecting more than one item at the time. What would the other mentioned shortcuts do in the context of our file dialog?

**Comment by alexballas** [comment]
Indeed, for this issue I agree that it does not make sense to have it. Multi file selection will allow however to introduce additional file operations as part of the file dialog like copy, move, delete in combination with folder creation.

**Comment by alexballas** [comment]
Actually I just checked "zenity --file-selection" and it does not support any of those. So maybe just ignore my last comments.
Thanks

**Comment by andydotxyz** [comment]
The ctrl+A, and shift/ctrl click will be added when we add multiple-selection, which will be a special mode for file opening.
